loader.qml

```diff
@@ -1,57 +1,193 @@
 import QtQuick 2.15
 import QtGraphicalEffects 1.12
 import QtQuick.Layouts 1.2
-import QtQuick.Controls 1.4
+import QtQuick.Controls 2.15
 import QtQuick.Dialogs 1.1
 
 ApplicationWindow {
     id: root
+    visible: true
+    minimumWidth: 640
+    minimumHeight: mysize + header.height // + menuBar.height
     property bool displayAmbient: false
     property bool round: true
+    property int mysize: 640
```

Comment:
Variables are named in camelCase, so that would be mySize.
But mySize isn't a great name. The "my" prefix always reminds me of entry-level tutorials where people create myVariable to make it seem a little bit funny. size would be just as descriptive.
Same for myFrame. Who does it belong to?

----

Also it's very confusing that it's defined in the scope of root because it's not even root's size, it's actually myFrame's size. I think that myFrame should be named previewArea, and this should be previewArea.size.
Or, it could also just be that you bind previewArea.height to previewArea.width and you just need to change previewArea.width, but I have no strong opinion about this.